File: 프로그래머스/unrated/181894. 2의 영역/2의 영역.py
def solution(arr):
    first_index = None
    last_index = None
    for index, value in enumerate(arr):
        if value == 2:
            if first_index is None:
                first_index = index
            last_index = index
    if first_index is None:
        return[-1]
    return arr[first_index:last_index+1]

File: 프로그래머스/0/181894. 2의 영역/2의 영역.java
import java.util.*;
class Solution {
    public int[] solution(int[] arr) {
        // 2가 모두 포함된 가장 작은 연속된 부분배열
        // Arrays.asList(arr)
        // 먼저 2가 포함된 INDEX를 찾음
        // MIN INDEX == START 이고
        // MAX INDEX == END 임
        int min = 100000, max=0;
        for (int i=0;i<arr.length;i++){
            if(arr[i]==2){
                min=Math.min(min,i);
                max=Math.max(max,i);
            }
        }
        if(min<=max){
            return Arrays.copyOfRange(arr,min,max+1);
        }else{
            return new int[]{-1};
        }
    }
}